### DISCUSSION TITLE
Loading of directus app freeze. API & APP running locally in debug mode.

### DISCUSSION THREAD

**Discussion by vencax** [directus/directus#17384] [Archive]
### Describe the Bug

After opening browser i got this in vite log:
`
3:53:42 PM [vite] warning: 
/home/vencax/prg/experiments/directus2/app/src/extensions.ts
22 |  export async function loadExtensions() {
23 |    try {
24 |      customExtensions = import.meta.env.DEV ? await import("@directus-extensions") : await import(`${getRootPath()}extensions/sources/index.js`);
   |                                                                                                   ^
25 |    } catch (err) {
26 |      console.warn(`Couldn't load extensions`);
The above dynamic import cannot be analyzed by Vite.
See https://github.com/rollup/plugins/tree/master/packages/dynamic-import-vars#limitations for supported dynamic import formats. If this is intended to be left as-is, you can use the /* @vite-ignore */ comment inside the import() call to suppress this warning.

  Plugin: vite:import-analysis
  File: /home/vencax/prg/experiments/directus2/app/src/extensions.ts

`
and loading of js files freezes and the page never loads. 

**If I try i in Firefox, it works.**

### To Reproduce

Following [Running locally](https://docs.directus.io/contributing/running-locally.html) how-to, having this  .env file:
`
ADMIN_EMAIL=vencax@********.cz
ADMIN_PASSWORD=h**es****7

KEY='directus2'
SECRET='afjslkfjal'

DB_FILENAME=db.sqlite
DB_CLIENT=sqlite3

LOG_LEVEL=debug
`

Running api and app separately to avoid race conds. 

### What version of Directus are you using?

9.22.4 (commit 21f6f3d0089142850dde1ad0b33c84d3da3d2cff)

### What version of Node.js are you using?

18.12.1

### What database are you using?

"sqlite3": "5.1.4",

### What browser are you using?

Google Chrome Version 108.0.5359.124 (Official Build) (64-bit)

### How are you deploying Directus?

running locally

**Comment by paescuj** [comment]
> **If I try i in Firefox, it works.**

Which is the other browser then?

**Reply by vencax** [reply]
The browser that do not finish loading app is Google Chrome. But I tested on my android phone with google chrome, opera. Both not working.

New info: when I run it on Firefox, it works (as I mentioned), but the error appears as well. So I suppose the error is not connected with the browser freeze?

I work on Linux machine (maybe worth mentioning)
